.cspell/misc-terms.txt

```diff
@@ -0,0 +1,4 @@
+colour
+colours
```

Comment:
@aloisklink,

I can change the language to `en-GB,en-US` to capture both spelling variants. Do you have a link to a hunspell `en-GB-oxendict` dictionary? I didn't find a clear source for one.

Reply:
done.

Reply:
Hmmmm, maybe it's not supported by hunspell: https://en.wikipedia.org/wiki/Oxford_spelling (or hunspell's `en-GB` already includes it in their dictionary). It's the variant of English used by ISO/ the UN, and most British academic journals.

It's basically `en-GB`, except most words use the `-ize` instead of `-ise`, e.g. organize vs organise. `en-GB,en-US` though should cover it completely, though!